src/server/sams/commands/delete_elastic_index.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8; -*-
+#
+# This file is part of SAMS.
+#
+# Copyright 2020 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from flask import current_app as app
+from eve_elastic import get_es
+from elasticsearch import exceptions as es_exceptions
+
+from superdesk import Command, command
+
+from sams.logger import logger
+
+
+class DeleteElasticIndex(Command):
+    """Delete elastic index
+
+    It removes all elastic indices.
+
+    Example:
+    ::
+
+        $ python -m sams.manage app:delete_elastic_index
+
+    """
+
+    def run(self):
+        logger.info('Delete elastic indices')
+        self.delete_elastic(app.config['ELASTICSEARCH_INDEX'])
+
+    @classmethod
+    def delete_elastic(cls, index_prefix):
+        """Deletes elastic indices with `index_prefix`
+
+        :param str index_prefix: elastix index
+        :raise: SystemExit exception if delete elastic index response status is not 200 or 404.
+        """
+
+        es = get_es(app.config['ELASTICSEARCH_URL'])
+        indices = list(es.indices.get_alias('{}_*'.format(index_prefix)).keys())
+
+        for es_resource in app.data.get_elastic_resources():
+            alias = app.data.elastic._resource_index(es_resource)
+            for index in indices:
+                if index.rsplit('_', 1)[0] == alias:
+                    try:
+                        logger.info('Removing elastic index "{}"'.format(index))
```

Comment:
I would probably log via `logger.info('Removing elastic index "%s"', index)`, with format it seems a bit complicated to me,
we should probably pick one or the other to be consistent, any opinion @ride90 @jerome-poisson ?

Reply:
I think `format` is a new style and `%` is old, so I would go with the `format`. 
Starting from python3.6 we can use **f-strings**, which are faster and nicer.

Reply:
it's not about using `%`, logger supports that message format

Reply:
ahh, right. 
In this case `logger.info('Removing elastic index "%s"', index)` seems more simple.

Are we support python 3.5 for SAMS? If not f-strings is the way to go imo.

Reply:
SAMS uses Superdesk-Core/develop as the application framework, so I would say Python 3.6+ are the supported versions.
Good point @petrjasek, I didn't realise `logger` had `vargs` capability, will update accordingly.